index.js

```diff
@@ -243,6 +246,58 @@ module.exports = function(version, _options) {
             }
 
             return output;
+        },
+        getBestMatchingLanguage: function(language) {
+            if (languages.instructions[language]) return language;
+
+            // If the language is not found, try a little harder
+            var splitLocale = language.toLowerCase().split('-');
+            var languageCode = false;
+            var scriptCode = false;
+            var countryCode = false;
+
+            if (splitLocale.lenth === 1) {
+                languageCode = splitLocale[0];
```

Comment:
Might as well set `languageCode` unconditionally at this point. If `language` is something bogus like `c++`, then we’ll end up falling back to English anyways.

Reply:
I'm going to keep the splitting code and the return code separated.